test/controllers/comments_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
```

Comment:
Remove tests, please. We are using Rspec and tests should be in another place and another format.